android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -62,14 +62,15 @@ public void startWithOptions(final PluginCall capOptions) {
         SentryAndroid.init(
```

Comment:
side note: bump the Android SDK to the latest GA, 5.0.0

Reply:
Sounds good, after this PR I'll make one to bump Android and another to bump JS